lib/reporters/json-reporter.js

```diff
@@ -34,7 +34,7 @@ var branchCoverage = function(fileData) {
     for (var j = 0; j < branches.length; j++) {
       if (_.isArray(branches[j])) {
         for (var k = 0; k < branches[j].length; k++) {
-          if (_.isArray(branches[j][k])) {
+          if (_.isObject(branches[j][k])) {
```

Comment:
Why this change? It seems like `branches[j][k]` is being treated as an array on line 39...